### PR TITLE
MINOR: [Website] Fix a typo in arrow-parquet-encoding-part-2

### PR DESCRIPTION
**Pull request by Ted-Jiang**

### CODE REVIEW

File: _posts/2022-10-08-arrow-parquet-encoding-part-2.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ A definition level of `0` would imply a null at the level of `d`:
 }
 ```
 
-A definition level of `1` would imply a null at the level of `d`
+A definition level of `1` would imply a null at the level of `d.d2`
```
Comment (Ted-Jiang) [inline _posts/2022-10-08-arrow-parquet-encoding-part-2.md:152]: @alamb @tustvold   i think this should be `d.d2`

Comment (alamb) [reply _posts/2022-10-08-arrow-parquet-encoding-part-2.md:152]: Yes I think you are correct

**Review by alamb — APPROVED** [review]
Thanks @Ted-Jiang  -- I would like  @tustvold to confirm as well

File: _posts/2022-10-08-arrow-parquet-encoding-part-2.md (hunk 1 above)
Comment (tustvold) [reply _posts/2022-10-08-arrow-parquet-encoding-part-2.md:152]: Yes we should probably also change the example to be valid JSON

```
"d": { }
```

**Review by tustvold — APPROVED** [review]

**Review by alamb — APPROVED** [review]
Thanks @Ted-Jiang  and @tustvold

**Pull request merged**